### PR TITLE
task 4

### PR DESCRIPTION
**Pull request by Limurezzz**
Latest deployed version

https://5r1qt1utef.execute-api.eu-west-1.amazonaws.com/dev/products
https://5r1qt1utef.execute-api.eu-west-1.amazonaws.com/dev/products/47d35977-1803-4332-98da-dac570cb2af7

### CODE REVIEW

**Comment by sleepwalky** [comment]
@Limurezzz тут на 1 балл?

**Comment by MaxGaranin** [comment]
Да, не очень понятно, что тут проверять. Будет доделка задания?

File: product-service/serverless.yml [hunk 1]
```diff
@@ -46,9 +42,13 @@ provider:
...
-# you can define service wide environment variables here
-#  environment:
-#    variable1: value1
+
+environment:
+  PG_HOST: database-1.cl66ht8caoqs.eu-west-1.rds.amazonaws.com
+  PG_PORT: 5432
+  PG_DATABASE: postgres
+  PG_USERNAME: postgres
+  PG_PASSWORD: oxxmlOy5wkL8MXSjd8Bd
```
Comment (skyRoma) [inline product-service/serverless.yml:51]: !Important Don't commit this to your repo. It is not safe for your money.

**Review by skyRoma — APPROVED** [review]

File: product-service/serverless.yml (hunk 1 above)
Comment (skyRoma) [reply product-service/serverless.yml:51]: FYI: Approved it as 1 point task

**Comment by Limurezzz** [comment]
@sleepwalky @MaxGaranin @skyRoma
Сегодня до вечера все доделаю!

**Pull request merged**